Eatery/Utils/Sort.swift

```diff
@@ -110,7 +110,7 @@ struct Sort {
         return eatery.sorted(by: sortByHoursClosure)
     }
 
-    //HelperFunction to get meal
+    // Helper function to get meal
```

Comment:
Inconsistent capitalization of first letter (e.g. see line 103). It's really not a big deal though. I would just ignore it for this PR.